### ISSUE TITLE
driveItem: preview api   getUrl  iphone12 pdf file can't search

### ISSUE THREAD

**Issue by Young-Young-Young** [microsoftgraph/msgraph-sdk-java#609]
we use driveItem: preview api  get "getUrl" ,when we use the url in iphone12 preview pdf file  we find we can’t use the search function.
ios os 14.2.1

**Comment by Young-Young-Young** [comment]
https://user-images.githubusercontent.com/13057846/103973607-bc471f80-51aa-11eb-920d-92d616d7c14a.mp4

**Comment by baywet** [comment]
Hi @Young-Young-Young 
Thanks for reaching out.
This repository is dedicated to issues with the java/android SDK for Microsoft Graph. I'm guessing that if you developed an iOS application/integration, you are probably using the [Objective-C](https://github.com/microsoftgraph/msgraph-sdk-objc) SDK instead?
Can you confirm? if so, I'll transfer your issue to the right repository.
Thanks

**Comment by Young-Young-Young** [comment]
@baywet  
use Java sdk , use driveItem: preview api get "getUrl",then we use the url in webview or mobile browser。

**Comment by baywet** [comment]
@Young-Young-Young thanks for the additional information. The preview URL is provided by the service, so is the preview "document" as the [documentation](https://docs.microsoft.com/en-us/graph/api/driveitem-preview?view=graph-rest-1.0) indicates.
This repository is dedicated to issues about the SDK itself, as this behavior is driven by the service, and not the SDK, there's not much we can do here.
I suggest you reopen your issue using the "feedback" section of the documentation page I provided the link to. This new issue will get transferred to the service team who might be able to help you.
Thanks!
Closing
